### PR TITLE
Bumps MongoDB.Driver from 2.13.2 to 2.28.0.

### PR DESCRIPTION
**Pull request by oriolaakeem**
Bumps [MongoDB.Driver](https://github.com/mongodb/mongo-csharp-driver) from 2.13.2 to 2.28.0.

### CODE REVIEW

**Comment by SteAmeR** [comment]
Any progress?

**Comment by vova3211** [comment]
thanks for contr, on it

File: src/AspNetCore.Identity.Mongo/Migrations/Schema6Migration.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
+﻿using MongoDB.Driver;
```
Comment (vova3211) [inline src/AspNetCore.Identity.Mongo/Migrations/Schema6Migration.cs:1]: what the point of this Migration? Seems like copy of `Schema5Migration`

**Review by vova3211 — COMMENTED** [review]
overall look fine, but for future:
1. please do not change package version inside PR. There could be multiple PRs, features etc. released simultaneously
2. please describe each fix/improvement that you made in the PR description

**Comment by oriolaakeem** [comment]
Noted

**Comment by tbureck** [comment]
Hey there, any reason you closed this PR?

Best regards

**Comment by oriolaakeem** [comment]
I assumed this had been re-implemented by @vova3211

**Comment by vova3211** [comment]
@oriolaakeem yes, exactly. Thanks for contribution anyway

@tbureck [latest release](https://www.nuget.org/packages/AspNetCore.Identity.Mongo) contains all changes made in this pr

**Pull request closed without merging**